### PR TITLE
Fix: Country flag not showing up

### PR DESCRIPTION
**Pull request by nonlooped**
The old website for country flags is down (I have no idea why) but this fixes it.

### CODE REVIEW

File: src/interactions/info/Weather.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class WeatherInfoCommand extends SubCommand {
     const windDeg = this.getWindDirection(data.wind.deg, lang);
 
     const country = data.sys.country.toString();
-    const flag = `https://www.countryflags.io/${country}/flat/64.png`;
+    const flag = `https://flagcdn.com/256x192/${country.toLowerCase()}.png`;
```
Comment (casperiv0) [inline src/interactions/info/Weather.ts]: I'd rather use the `original` version than `waving`

```suggestion
    const flag = `https://flagcdn.com/w40/${country.toLowerCase()}.png`;
```

**Review by casperiv0 — CHANGES_REQUESTED** [review]

File: src/interactions/info/Weather.ts (hunk 1 above)
Comment (nonlooped) [reply src/interactions/info/Weather.ts]: I will change it right now!

File: src/interactions/info/Weather.ts [hunk 2]
```diff
@@ -53,7 +53,8 @@ export default class WeatherInfoCommand extends SubCommand {
...
```
Comment (casperiv0) [inline src/interactions/info/Weather.ts]:
```suggestion
```

**Review by casperiv0 — APPROVED** [review]
Thanks!

**Comment by nonlooped** [comment]
> Thanks!

You're welcome!

**Comment by nonlooped** [comment]
Didn't mean to close it

**Pull request merged**